Configure

```diff
@@ -545,18 +554,18 @@ my %user = (
     AS          => undef,
     ASFLAGS     => [],
     CC          => env('CC'),
-    CFLAGS      => [],
+    CFLAGS      => [ env('CFLAGS') || () ],
```

Comment:
Feature creep?

Reply:
Uhmmm, no?  Or how do you mean?

Reply:
Adding the cflags environment read?  that's new isn't it?

Reply:
I can make this PR master-only, if you prefer.

Reply:
I'm okay with this change going to 1.1.1.  It makes sense.

Reply:
Okie. The added CPPFLAGS is *needed*, and my thinking is that if I add support for that, it would be weird to leave the other "classic" variables out...